.github/workflows/backport.yml

```diff
@@ -7,6 +7,7 @@ on:
 
 jobs:
   backport:
+    if: ${{ contains(github.event.label.name, 'backport') }}
```

Comment:
The workflow also needs updated parameters. You can find the correct implementation here: https://github.com/opensearch-project/OpenSearch/blob/main/.github/workflows/backport.yml#L36